CHANGELOG.md

```diff
@@ -117,7 +117,7 @@ The [`logout`](https://developer.reach5.co/api/identity-web/#logout) method was
 ### 02/12/2019
 
 ### Fixes
-Add the missing implementation of `Buffer` to address an issue with the UMD bundle when PKCE is enabled.
+Add the missing implementation of `Buffer` to address an issue with the UMD bundle when PKCE is enforced.
```

Comment:
Back in early 2019, PKCE actually was either enabled or disabled. The change to "enforce" it came much later.